### DISCUSSION TITLE
Documentation: Please give full examples for mkv workflow

### DISCUSSION THREAD

**Discussion by sgofferj** [quietvoid/dovi_tool#327] [General]
I'm not exactly an expert in video codecs and file formats - and probably I'm not alone with this. I would like to suggest/request you add a few complete examples for full mkv workflows, e.g. how to "convert" an mkv file with video track e.g. DV profile 5 to an mkv file with video track HDR10.

**Comment by quietvoid** [comment]
This is not a video conversion tool.
Even your example is way out of scope.
